hazel.bzl

```diff
@@ -56,6 +56,13 @@ haskell_import(
     package = "{pkg}",
     visibility = ["//visibility:public"],
 )
+# Cabal packages can depend on other Cabal package's cbits, for example for
+# CPP includes. To enable uniform handling we define a `-cbits` target for
+# every Hazel Haskell target. In case of core_libraries this is just a dummy.
```

Comment:
Are there any alternatives to defining these dummy targets?

Reply:
None that I'm aware of. If we don't define a `-cbits` targets for every package (include core-packages), then we'd have to somehow introduce conditional dependencies on the use-site. I don't know how to do that. E.g. if package `A` depends on `B`, how would `A` know if it should depend on `B-cbits` or if `B-cbits` doesn't exist?

I think there's actually an example of a core-package that could have a proper, non-dummy, `-cbits` target. Namely, `unix`. Currently, there's [special casing](https://github.com/FormationAI/hazel/blob/87f0f801c0276b2eceaa1eed7132838d99e47e07/third_party/cabal2bazel/bzl/cabal_package.bzl#L41) in Hazel to include the unix headers whenever a package depends on `unix`. However, if the `unix` core-package's `-cbits` would include those headers, then that special casing code could be removed.

I haven't figured out how to include those headers into the `unix` package properly. It's a bit tricky, because the unix headers come from the ghc-workspace. This also seemed out of scope for this PR.